src/components/dropdown/UIDropdown.vue

```diff
@@ -16,7 +16,7 @@
              :style="dropdownStyle"
              role="group"
              @click.stop>
-            <slot />
+            <slot :toggle="toggle" :show="show" :hide="hide" />
```

Comment:
Reckon lets also set the expose attribute for the sake of consistency

Reply:
👍